using_openshift/cli.adoc

```diff
@@ -91,9 +91,9 @@ Multiple `.kubeconfig` files are allowed; they are loaded at runtime and merged
 .  The `.kubeconfig` file in the current directory
 .  The `.kubeconfig` file inside the `.kube` directory in the user's home: `~/.kube/.kubeconfig`
 
-For more details about `.kubeconfig` files, see link:kubeconfig.adoc[].
+For more details about `.kubeconfig` files, see link:../using_openshift/kubeconfig.html[Kubernetes].
```

Comment:
Per the above comment in the `_build_cfg.yml` file, just a reminder to also update this line if the "Kubernetes" title is indeed changed.